learningequality/ka-lite
# Open access control panel

Depends on #990, #991, #995 , solves issue #994

Allow control panel on distributed server to be visible without being on a zone.

Testing:
- Install, remove zone (through `./manage.py shell_plus`, by running `Zone.objects.all().delete()`
- Go to easy_admin page
- Click the "network management" page.
- Explore!  Should work as it does with a zone.
